.buildkite/pipeline.yml

```diff
@@ -0,0 +1,40 @@
+env:
+  BK_QUEUE_NAME: "testqueue" # probably don't need this
+  TESTER_VAULT_VERSION: "1.11.2" # this probably needs to get bumped over time
+  SVC_VAULT_VERSION: "1.11.2" # this probably needs to get bumped over time
+  VAULT_ADDR: "http://vault-svc:8200"
+  BUILDKITE_PIPELINE_SLUG: "my_pipeline"
+  DUMP_ENV: "true"
+  VAULT_DEV_ROOT_TOKEN_ID: "88F4384B-98E9-4AE3-B00C-F55678F89080"
```

Comment:
if they are only used in a single step, why not add them on that step instead?

Reply:
I tried defining the values a few different ways (most of these values are also defined or have defaults in the docker-compose) with varying degrees of success. Ended up only leaving `VAULT_DEV_ROOT_TOKEN_ID` and `VAULT_ADDR` as globally defined, the rest are defined in the plugin env directly.